src/L6/datasorter_test.py

```diff
@@ -1,16 +1,43 @@
 import random
 import unittest
+import os
 
-from L6.datasorter import DataSorter
+from datasorter import DataSorter
+from datasorter import NotCSVException
 
 
 class DataSorterTest(unittest.TestCase):
 
     def setUp(self):
         self.under_test = DataSorter()
+        file = open("test1.txt", "w+")
+        file.write("1")
+        file.close()
+        file = open("test2.csv", "w+")
+        file.write("1, 2, 3")
+        file.close()
+
+    def test_set_input_data_with_invalid_paths(self):
+        with self.assertRaises(TypeError):
+            self.under_test.set_input_data(48)
+        with self.assertRaises(FileNotFoundError):
+            self.under_test.set_input_data("test0.csv")
+
+        with self.assertRaises(NotCSVException):
+            self.under_test.set_input_data("test1.txt")
+
+    def test_set_input_data_with_valid_paths(self):
+        self.assertTrue(self.under_test.set_input_data("test2.csv"))
+        self.assertIsNotNone(self.under_test.data)
+        self.assertIsInstance(self.under_test.data, list)
+        self.assertEqual(3, len(self.under_test.data))
+        self.assertEqual('3', self.under_test.data[2])
```

Comment:
We are not supporting `str` elements per the professors requirements. We only support int and/or float